letsencrypt-auto

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 #
-# Download and run the latest release version of the Let's Encrypt client.
+# Download and run the latest release version of the Certbot client.
```

Comment:
All changes to this file should be reverted. It should remain the same as the file from our last release.